### ISSUE TITLE
The desired archetype does not exist (com.adobe.cq.spa.archetypes:aem-spa-project-archetype:1.0.5-SNAPSHOT)

### ISSUE THREAD

**Issue by davesag** [adobe/aem-spa-project-archetype#110]
## Bug Report

**Current Behavior**

I have been following the instructions in the readme, cloned this repo, ran `mvn clean install archetype:update-local-catalog` and then `mvn archetype:crawl`, then created a new folder and ran

```
mvn archetype:generate \
 -DarchetypeCatalog=internal \
 -DarchetypeGroupId=com.adobe.cq.spa.archetypes  \
 -DarchetypeArtifactId=aem-spa-project-archetype  \
 -DarchetypeVersion=1.0.5-SNAPSHOT
```

and it failed with

```
[INFO] Generating project in Interactive mode
[WARNING] Archetype not found in any catalog. Falling back to central repository.
[WARNING] Add a repository with id 'archetype' in your settings.xml if archetype's repository is elsewhere.
[WARNING] The POM for com.adobe.cq.spa.archetypes:aem-spa-project-archetype:jar:1.0.5-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  18.945 s
[INFO] Finished at: 2019-10-15T09:53:36+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-archetype-plugin:3.1.2:generate (default-cli) on project standalone-pom: The desired archetype does not exist (com.adobe.cq.spa.archetypes:aem-spa-project-archetype:1.0.5-SNAPSHOT) -> [Help 1]
```

**Expected behavior/code**

I expected a new project to be successfully generated

**Environment**
- AEM version and patch level (e.g. AEM 6.5 installed via docker - see https://github.com/pawangonnakuti/aem-author/blob/master/Dockerfile)
- AEM SPA Archetype `1.1.0`
- JRE version `Java(TM) SE Runtime Environment (build 1.8.0_92-b14)`

**Possible Solution**

No idea

**Additional context / Screenshots**

None

**Comment by davesag** [comment]
I got this to work by using the following command

```
mvn archetype:generate \
 -DarchetypeCatalog=local \
 -DarchetypeGroupId=com.adobe.cq.spa.archetypes  \
 -DarchetypeArtifactId=aem-spa-project-archetype  \
 -DarchetypeVersion=1.1.0
```

I recommend the README be updated to reflect this

**Comment by samuelmeuli** [comment]
Thanks for reporting! This will be resolved with 82e3a6b5fcd5e65ec0d6511379c3bdec3811117c.
